packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/unitSpecs/collectingLocation/index.js

```diff
@@ -27,7 +27,8 @@ const parts = [
   {
     componentName: 'LocalityDropdownPickerSearch',
     componentProps: {
-      columnProps: { width: 10 },
+      columnProps: { width: 11 },
+      fluid: true,
```

Comment:
is this really necessary?

Reply:
Yes of course, but my comment refers to `fluid: true`. I think the dropdown should fill the column anyway.